radixdlt-engine/src/main/java/com/radixdlt/atom/Atom.java

```diff
@@ -58,7 +58,7 @@
 	@DsonOutput({Output.ALL})
 	private final ECDSASignature signature;
 
-	private final ImmutableList<REInstruction> instructions;
+	private final List<REInstruction> instructions;
```

Comment:
I really like this change. I think we should get rid of ImmutableList/Map/Set/etc. in field and parameter declarations.